core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -3072,24 +3072,33 @@ private SqlDialect nonOrdinalDialect() {
     sql(query).dialect(HiveSqlDialect.DEFAULT).ok(expected);
   }
 
-  @Test void testMysqlCastToBigint() {
```

Comment:
+1

Reply:
> minor: recommended to add new unit tests and annotate corresponding jira link.

Thank you for your suggestion. I have added corresponding jira link comments, but I think this is the same type conversion issue. It would be better to handle different dialect specificity in one test case, otherwise there may be many redundant test cases